lib/server-impl.test.js

```diff
@@ -99,3 +107,13 @@ test('should not create a server using create()', async t => {
     });
     t.true(typeof server === 'undefined');
 });
+
+test.only('should shutdown the server when calling stop()', async t => {
```

Comment:
This `.only` looks like an artefakt from local testing that shouldn't be merged, or should it?

Reply:
You are right sorry about that. I opened another PR to fix that https://github.com/Unleash/unleash/pull/674

Thanks !